rotkehlchen/inquirer.py

```diff
@@ -458,10 +461,18 @@ def _query_oracle_instances(
                 continue
 
             try:
-                price = oracle_instance.query_current_price(
-                    from_asset=from_asset,  # type: ignore  # type is guaranteed by the if above
-                    to_asset=to_asset,  # type: ignore  # type is guaranteed by the if above
-                )
+                # Using this instead of isinstance() to don't get cicrcular imports
+                if oracle_instance.__class__.__name__ == 'ManualCurrentOracle':
```

Comment:
you probably don't need that as I said this argument is not required

Reply:
Sorry, by "this argument" what did you mean?

Reply:
I meant the `match_main_currency` differentiation but we addressed this in another comment.

I still don't like this approach though. It's gonna be reverted anyway but do you think you can do it differently?

Can't you add the `match_main_currency` argument to all the oracles and just ignore it or add **kwargs to all theoracles and only use it for manual one?

We can probably avoid this extra if check this way, right?